### PR TITLE
OCPBUGS-11899-RE: Updated Creating the K8S manifest and Ignition conf…

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-11899](https://issues.redhat.com/browse/OCPBUGS-11899)

Version(s):
4.14 through to 4.12

Link to docs preview:
* [Creating the Kubernetes manifest and Ignition config files (AWS)](https://64288--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws#installation-user-infra-generate-k8s-manifest-ignition_installing-restricted-networks-aws)
* [Creating the Kubernetes manifest and Ignition config files (Azure)](https://64288--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra#installation-user-infra-generate-k8s-manifest-ignition_installing-azure-user-infra)
* [Creating the Kubernetes manifest and Ignition config files (Azure Stack Hub)](https://64288--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra#installation-user-infra-generate-k8s-manifest-ignition_installing-azure-stack-hub-user-infra)

QE review:
- [x] QE approval on the JIRA

Additional information:

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

File: modules/installation-user-infra-generate-k8s-manifest-ignition.adoc [hunk 1]
```diff
@@ -159,26 +159,26 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_master-machi
...
 . Optional: If you do not want the cluster to provision compute machines, remove
 the Kubernetes manifest files that define the worker machines:
 endif::user-infra-vpc[]
 endif::gcp[]
-ifdef::aws,azure,ash,user-infra-vpc[]
+ifdef::azure,ash,user-infra-vpc[]
 . Remove the Kubernetes manifest files that define the worker machines:
-endif::aws,azure,ash,user-infra-vpc[]
+endif::azure,ash,user-infra-vpc[]
 ifdef::aws,azure,ash,gcp[]
```
Comment (michaelryanpeter) [inline modules/installation-user-infra-generate-k8s-manifest-ignition.adoc:182]: @dfitzmau For AWS, was it intentional to include the second CLI command block for step one and remove the lead-in wording? I am wondering if the `ifdef` on line 182 should remove `aws`.

 AWS is on the left and Azure is on the right:
Screenshot from 2023-09-19 13-22-44

**Review by michaelryanpeter — COMMENTED** [review]
I think there might be small error with one of the `ifdef`s. Please feel free to reach out to me directly if you want to discuss or when you are ready for a merge.

File: modules/installation-user-infra-generate-k8s-manifest-ignition.adoc (hunk 1 above)
Comment (dfitzmau) [reply modules/installation-user-infra-generate-k8s-manifest-ignition.adoc:182]: I'll ask the SME to chime in here as some input on a thread lead to some confusion around due 4.14 updates, which are outside the scope of my PR, as my PR corrects existing content.

File: modules/installation-user-infra-generate-k8s-manifest-ignition.adoc [hunk 2]
```diff
@@ -159,26 +159,26 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_master-machi
...
 ifdef::gcp[]
 ifndef::user-infra-vpc[]
 . Optional: If you do not want the cluster to provision compute machines, remove
 the Kubernetes manifest files that define the worker machines:
 endif::user-infra-vpc[]
 endif::gcp[]
-ifdef::aws,azure,ash,user-infra-vpc[]
+ifdef::azure,ash,user-infra-vpc[]
 . Remove the Kubernetes manifest files that define the worker machines:
-endif::aws,azure,ash,user-infra-vpc[]
```
Comment (patrickdillon) [inline modules/installation-user-infra-generate-k8s-manifest-ignition.adoc:181]: Azure looks correct, but AWS is missing some pieces.  We need to have all 3, as Azure does:

1. Remove the Kubernetes manifest files that define the control plane machines:

1. Remove the Kubernetes manifest files that define the control plane machine set:

1. Remove the Kubernetes manifest files that define the worker machines:


Right now, AWS only has (part of) Remove the Kubernetes manifest files that define the worker machines:

Comment (dfitzmau) [reply modules/installation-user-infra-generate-k8s-manifest-ignition.adoc]: Thanks for clarifying, Patrick. I added these steps back in. Additionally, I included an additional step after the `grep` command that now addresses the docs bug Jira update.

File: modules/installation-user-infra-generate-k8s-manifest-ignition.adoc [hunk 3]
```diff
@@ -435,7 +436,11 @@ $ grep "release.openshift.io/feature-set" *
 ----
 // Right now, only the CAPI Operator is an issue, but it might make sense to update `0000_30_capi-operator_00_credentials-request.yaml` to `<tech_preview_credentials_request>.yaml` for the future.
 
-.. Create a `cco-configmap.yaml` file in the manifests directory with the Cloud Credential Operator (CCO) disabled:
+.  Based on the output from the `grep` command in the previous step, delete all `CredentialsRequest` objects that have the `TechPreviewNoUpgrade` annotation.
```
[the remark this answers is not in the document]
Comment (gpei) [reply modules/installation-user-infra-generate-k8s-manifest-ignition.adoc]: @dfitzmau The new words looks good to me, thanks for the update!

File: installing/installing_aws/installing-restricted-networks-aws.adoc [hunk 4]
```diff
@@ -98,8 +98,13 @@ include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
 //include::modules/installation-three-node-cluster.adoc[leveloffset=+2]
 
+// Creating the Kubernetes manifest and Ignition config files
 include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+* xref:../../installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc#manually-create-iam_installing-restricted-networks-aws-installer-provisioned[Manually creating long-term credentials]
```
Comment (kcarmichael08) [inline installing/installing_aws/installing-restricted-networks-aws.adoc:106]: The 4.13 cherry pick seems to be barfing on this reference.

Comment (kalexand-rh) [reply installing/installing_aws/installing-restricted-networks-aws.adoc:106]: @dfitzmau, will you PTAL? You're probably going to need to open manual cherry-picks to address this issue.

Comment (dfitzmau) [reply installing/installing_aws/installing-restricted-networks-aws.adoc:106]: Yes, of course. 

I raised:

- https://github.com/openshift/openshift-docs/pull/66072/files
- https://github.com/openshift/openshift-docs/pull/66073/files

File: installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc [hunk 5]
```diff
@@ -77,7 +77,14 @@ include::modules/installation-azure-stack-hub-config-yaml.adoc[leveloffset=+2]
 
 include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 include::modules/installation-user-infra-exporting-common-variables-arm-templates.adoc[leveloffset=+2]
+
+// Creating the Kubernetes manifest and Ignition config files
 include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-create-iam_manually-creating-iam-azure[Manually creating long-term credentials]
```
Comment (jeana-redhat) [inline installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc:86]: Found this while working on some related content. Do we want a link to global/normal/non-Stack-Hub Azure from the ASH docs here?

**Pull request merged**